man/utop.1

```diff
@@ -98,11 +98,115 @@ with options available for
 .BR ocaml (1).
 
 A commonly used option is
-\fB--require\fI package\fR
+\fB-require\fI package\fR
 to load \fIpackage\fR into the execution environment. It is equivalent
 to using \fb#require\fR from inside
 .BR utop (1).
 
+.PP
+.TP
+.BI -absname
+Show absolute filenames in error message
```

Comment:
Arguably there should be "."s at the end of sentences here. I recognize these aren't in the output of `utop --help` of course.